artman/cli/conductor.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Main class to start conductor.
+"""
+
+from logging import INFO
+import argparse
+import logging as pylog
+import sys
+
+from artman.conductors import cloudtasks_conductor
+from artman.utils.logger import logger, setup_logging
+
+def start(*args):
+    if not args:
+        args = sys.argv[1:]
+    queue_name, log_local = _parse_args(*args)
+    if log_local:
+        pylog.basicConfig()
+    cloudtasks_conductor.run(queue_name)
+
+def _parse_args(*args):
```

Comment:
From a design perspective, I would prefer return the `flags` object and let it be used by the result methods rather than return a tuple. That will require less maintenance if this has to be altered in the future.